index.bs

```diff
@@ -648,6 +648,23 @@ Additionally, implementing these items may also have a beneficial impact on the
 
 Issue: <a href="https://github.com/w3c/deviceorientation/issues/87">Further implementation experience</a> is being gathered to inform the limit for the maximum sampling frequency cap.
 
+Accessibility considerations {#a11y}
+===========================================================
+
+DeviceOrientation events provide opportunities for novel forms of input, which can open up novel interactions for users. In order to ensure that as many people as possible will be able to interact with the experiences you build, please consider the following.
+
+* It's important for alternative means of providing input to be available, so that people who can't make the required gestures have another way to interact - examples may include people with dexterity-related disabilities, or people who use eye gaze, or head-tracking input.
+    - For games, consider supporting either game controller, keyboard or mouse input in addition.
+    - For web apps, consider providing UI, such as a button, menu command, and/or keyboard shortcut, to perform the function.
+
+* It's essential that users can undo any accidental input - this may be particularly relevant for people with tremors.
```

Comment:
Also please with some grace period. I don't know how many times I try to swipe Gmail away on my phone and instead I swipe emails away and I cannot get them back or find out which one I removed

Reply:
I don't think we can include examples that mention specific products, but I've had similar experiences with all sorts of sites and apps - it's definitely something we want to encourage devs to think about (glad you agree on that).